common/persistence/sql/sqlplugin/postgresql/session/session.go

```diff
@@ -122,7 +125,6 @@ func buildDSNAttr(cfg *config.SQL) url.Values {
 			parameters.Set(sslMode, sslModeRequire)
 		} else {
 			parameters.Set(sslMode, sslModeFull)
-			parameters.Set(sslHost, cfg.TLS.ServerName)
```

Comment:
Why removing this line?

Reply:
Suppose, we have a connection string like `postgresql://user@some_host:port`
This line adds query parameter `host` and our connection string becomes `postgresql://user@some_host:port?host={TLS.ServerName}`.

Now, we have two `host` parameters:
First is from the host part of our connection string: https://github.com/lib/pq/blob/3d613208bca2e74f2a20e04126ed30bcb5c4cc27/url.go#L59
And another one from `?host=...` https://github.com/lib/pq/blob/3d613208bca2e74f2a20e04126ed30bcb5c4cc27/url.go#L71

Then these options are sorted lexicographically and the last one is taken https://github.com/lib/pq/blob/master/conn.go#L555

So, there is no need to have another `host` parameter.